docs/learning/howto/runner-service-windows.md

```diff
@@ -0,0 +1,210 @@
+[2023-10-04 13:12:51] [info]  ( prunsrv.c:2102) [ 5164] Apache Commons Daemon procrun finished.
+```
+
+7. Now press the Windows key + R key combination, then type `services.msc` and press the Enter key.
+
+8. Scroll down the Service list and locate the "Runner" service.
+
+![Launching services.msc](/assets/img/raas2.png)
+
+9. Click the right button and select "Start", after a couple of seconds, the service must be shown as "Running" status.
+
+![Windows Services](/assets/img/raas5.png)
+
+### How to remove the service
+
+To stop and remove the Automation Runner service check the following steps:
+
+1. Open a new CMD terminal with Administrative rights.
+
+2. Go to the `C:\runner\` folder.
+
+3. Execute `runner.exe //DS/Runner`. This process could take around 30 seconds.
+
+Now, the service is down and unregistered from Windows Services.
+
+## An alternative way: Installing the Automation Runner service through NSSM
+
+NSSM is a program used to register and manage services in Windows. It provides a user-friendly interface for creating and configuring Windows services without the need for complex command-line instructions. NSSM is particularly useful for running applications or scripts as services, ensuring that they start automatically with Windows, and can run in the background without user interaction. NSSM simplifies the process of working with Windows services, making it accessible for both experienced administrators and users who may not be familiar with the intricacies of service management on Windows.
+
+NOTE: NSSM is an inactive project, the last build was released in 2014.
+
+1. Download NSSM from this link.
+
+2. Uncompress the `.zip` file.
+
+3. Open a new PowerShell terminal with Administrator rights and go to the uncompressed NSSM folder.
+
+4. Execute `./nssm.exe install "Runner"`. A new window will appear.
+
+![NSSM Service Editor](/assets/img/raas8.png)
+
+5. On the "Path" field put the Java 11 JRE binary path (including the `java.exe` at the end).
+
+6. On the "Startup directory" put the Automation Runner folder, e.g. `C:\runner\`.
+
+7. On the Arguments field put `-jar runner.jar`, then click on the "Install service" button.
+
+8. Now press the `Windows key + R` key combination, type `services.msc`, and press the `Enter` key.
+
+9. Scroll down the Service list and locate the "Runner" service.
+
+10. Click the right button and select "Start", after a couple of seconds, the service must be shown as "Running" status.
+
+### How to remove the service
+
+1. Open a new PowerShell terminal with Administrator rights and go to the uncompressed NSSM folder.
+
+2. Stop the service with the following command: `nssm stop "Runner"`.
+
+3. Then, remove the service with the `nssm remove "Runner" confirm` command.
+
+The service is down and unregistered from Windows Services.
+
+## Resources
+
+[Automation Runner](https://docs.rundeck.com/docs/administration/runner/pre-4-11-runners.html#runner)
```

Comment:
Local links should go to the `.md` file using the path after `/docs/`. in this case:

`/administration/runner/pre-4-11-runners.md#runner`

Using a direct link doesn't allow it to work within the staged or historical version sites properly.

Reply:
Changing!